### PR TITLE
Decode url before fetching filemap

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: test/Microsoft.DocAsCode.Build.TableOfContents.Tests/TocDocumentProcessorTest.cs [hunk 1]
```diff
@@ -50,8 +51,42 @@ public override void Dispose()
...
+                }
+            };
+
+            AssertTocEqual(expectedModel, model);
+        }
+
+        [Fact]
         public void ProcessMarkdownTocWithAbsoluteHrefShouldSucceed()
         {
+            var file1 = _fileCreator.CreateFile(string.Empty, FileType.MarkdownContent, "a");
```
Comment (superyyrrzz) [inline test/Microsoft.DocAsCode.Build.TableOfContents.Tests/TocDocumentProcessorTest.cs]: >var file1 = _fileCreator.CreateFile(string.Empty, FileType.MarkdownContent, "a"); [](start = 12, length = 81)

remove...

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

File: test/Microsoft.DocAsCode.Build.TableOfContents.Tests/TocDocumentProcessorTest.cs (hunk 1 above)
Comment (vicancy) [inline test/Microsoft.DocAsCode.Build.TableOfContents.Tests/TocDocumentProcessorTest.cs]: ....

---
In reply to: [96348009](https://github.com/dotnet/docfx/pull/1188#discussion_r96348009) [](ancestors = 96348009)

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**